slamd/design_assistant/static/comment.js

```diff
@@ -31,14 +31,14 @@ export async function handleCommentSubmission() {
   const submit_comment_button = document.getElementById("submit_comment_button");
```

Comment:
In dem File muss der Import oben angepasst werden (siehe meine Anmerkungen im Chat)